cirrus/google_cloud/manager.py

```diff
@@ -44,6 +44,18 @@
     'STANDARD'  # alias for MULTI_REGIONAL/REGIONAL, based on location
 ]
 
+COMPUTE_ENGINE_DEFAULT = 'COMPUTE_ENGINE_DEFAULT'
```

Comment:
let's add a suffix `_SERVICE_ACCOUNT` and make these importable from top level `cirrus.google_cloud` module. e.g. add `from .manager import COMPUTE_ENGINE_DEFAULT`... to `cirrus.google_cloud.__init__`. then in fence you can do `if type == cirrus.google_cloud.COMPUTE_ENGINE_DEFAULT`